graylog2-server/src/main/java/org/graylog2/configuration/ExposedConfiguration.java

```diff
@@ -64,24 +63,6 @@
     @JsonProperty("allow_leading_wildcard_searches")
     public abstract boolean allowLeadingWildcardSearches();
 
-    @JsonProperty("rotation_strategy")
```

Comment:
Please update the UPGRADING file to document the changed `/system/configuration` response.